src/test/java/nl/tudelft/contextproject/util/SizeTest.java

```diff
@@ -50,7 +49,7 @@ public void testEqualsEqual() {
 	 */
 	@Test
 	public void testEqualsSame() {
-		assertTrue(size.equals(size));
+		assertEquals(size, size);

```

Comment:
While I completely agree with you, PMD is nagging us about using `assertTrue(x.equals(y))` instead of `assertEquals(x, y)`. To avoid this error, I think it is worth it to make the test slightly less clear.